doto-backend/src/routes/router.js

```diff
@@ -63,34 +63,22 @@ router.post('/schedule/post', function(req, res){
     task.save(function(err){
```

Comment:
hey, does this function handle when certain params are missing such as priority?

Reply:
I think there need to be some fixes to this function. It fails if a required parameter is missing, but also fails if a non-required param is missing. We also need to add status codes to the responses.

Reply:
Status responses have been added (200 for ok, 400 for fail), and I checked using postman and it doesn't fail unless a required parameter is missing. This API still passes if only the required parameters are there, and all the non-required parameters are missing.

Reply:
Sounds good. As long as it doesn't fail if a non-required parameter is missing.

Reply:
yeah that's true. because we have validations in our db schema for required fields.